common/app/common/metrics.scala

```diff
@@ -221,9 +221,14 @@ class CloudWatchMetricsLifecycle(
           s"${gc.name}-gc-count-per-min",
           "Used heap memory (MB)",
           StandardUnit.Count,
-          () => gc.gcCount.toLong,
+          () => gc.gcCount.toLong.toDouble,
+        ),
+        GaugeMetric(
+          s"${gc.name}-gc-time-per-min",
+          "Used heap memory (MB)",
+          StandardUnit.Count,
+          () => gc.gcTime.toLong.toDouble,
```

Comment:
`gc.gcTime` is already a double here. Would just removing the `toLong` work?

```suggestion
          () => gc.gcTime,
```

Reply:
Good catch! Thanks, updated the code.